GeorgianMemenator/Scenes/CatTagsList/CatTagsListViewModel.swift

```diff
@@ -8,11 +8,57 @@
 
 import SwiftUI
 import Foundation
+import Combine
 
 class CatTagsListViewModel: ObservableObject {
-    @Published var tags: [CatTagDetail] = [
-        CatTagDetail(name: "Sphynx cat"),
-        CatTagDetail(name: "Munchkin cat"),
-        CatTagDetail(name: "Bengal cat")
-    ]
+    private let catTagsService: CatListServing
+    
+    @Published var tags: [CatTag] = []
+    
+    @Published private(set) var hasError: Bool = false
```

Comment:
This pr does not concentrate on loading, errors or image loading. Please delete uneeded properties